.storybook/preview.js

```diff
@@ -0,0 +1,26 @@
+import { addDecorator, addParameters, configure } from "@storybook/svelte";
```

Comment:
Can you tell me about the choice to use svelte instead of `@storybook/html`? I'm not familiar with it - will it help us build out the examples more programmatically?

Reply:
yes exactly, more like react or angular although by default plain html5 is valid svelte file